client/elements/sc-page-dictionary.html

```diff
@@ -296,14 +296,40 @@ <h3>Similar Spelling</h3>
           }
 
           _glossaryLoaded() {
-              console.log("loaded Glossary");
-              console.log(this._getGlossaryDescription("dhamma"));
-              console.log(this.glossaryReturns);
+              let result = {};
+              for (let i=0; i<this.glossaryReturns[0].length; i++) {
+                  result[this.glossaryReturns[0][i].word] = this.glossaryReturns[0][i].text;
+              }
```

Comment:
`this.glossaryReturns[0].forEach((item) => result[item.word] = item.text);`

Reply:
Thanks!